src/rai_bench/rai_bench/tool_calling_agent/predefined/basic_tasks.py

```diff
@@ -0,0 +1,299 @@
+    expected_optional_args={"timeout_sec": int},
+)
+rosout_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/rosout"},
+    expected_optional_args={"timeout_sec": int},
+)
+joint_states_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/joint_states"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+# Odometry subtasks
+odom_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/odom"},
+    expected_optional_args={"timeout_sec": int},
+)
+filtered_odom_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/odometry/filtered"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+# Transform subtasks
+tf_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/tf"},
+    expected_optional_args={"timeout_sec": int},
+)
+tf_static_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/tf_static"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+
+# Robot description subtasks
+robot_description_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/robot_description"},
+    expected_optional_args={"timeout_sec": int},
+)
+robot_description_semantic_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/robot_description_semantic"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+# Sensor data subtasks
+scan_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/scan"},
+    expected_optional_args={"timeout_sec": int},
+)
+pointcloud_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/pointcloud"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+
+# Robot description subtasks
+robot_description_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/robot_description"},
+    expected_optional_args={"timeout_sec": int},
+)
+robot_description_semantic_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/robot_description_semantic"},
+    expected_optional_args={"timeout_sec": int},
+)
+
+# Sensor data subtasks
+scan_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/scan"},
+    expected_optional_args={"timeout_sec": int},
+)
+pointcloud_subtask = CheckArgsToolCallSubTask(
+    expected_tool_name="receive_ros2_message",
+    expected_args={"topic": "/pointcloud"},
+    expected_optional_args={"timeout_sec": int},
+)
```

Comment:
Some subtasks are repeated 3 times in this file.
Also, `robot_description_*` subtasks are never used. Are they needed?